test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -662,3 +653,48 @@ UCS_TEST_P(test_ucp_proto_mock_self, rkey_ptr)
 }
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_proto_mock_self, self, "self")
+
+class test_ucp_proto_mock_gpu : public test_ucp_proto_mock {
+public:
+    test_ucp_proto_mock_gpu()
+    {
+        mock_transport("rc_mlx5");
+    }
+
+    virtual void init() override
+    {
+        add_mock_iface("mock", [](uct_iface_attr_t &iface_attr) {
+            iface_attr.cap.am.max_short  = 2000;
+            iface_attr.bandwidth.shared  = 28e9;
+            iface_attr.latency.c         = 600e-9;
+            iface_attr.latency.m         = 1e-9;
+            iface_attr.cap.get.max_zcopy = 16384;
+        });
+        test_ucp_proto_mock::init();
+    }
+};
+
+UCS_TEST_P(test_ucp_proto_mock_gpu, cuda_managed_ppln_host_frag,
```

Comment:
btw, is it possible to add test for the changes in this PR? I.e. to check that cma can still be selected for rndv?
AFAIU, this test flow is not affected by the changes in this PR

Reply:
There is already a CMA test:
https://github.com/openucx/ucx/blob/12c907a9fa34df3c254c97fd72d3b6e23e905e36/test/gtest/ucp/test_ucp_proto_mock.cc#L565